bioblend/galaxy/tool_dependencies/__init__.py

```diff
@@ -75,7 +82,7 @@ def summarize_toolbox(
           the GUI and therefore is subject to breaking changes.
         """
         assert index_by in ["tools", "requirements"], "index_by must be one of 'tools' or 'requirements'."
-        params = {
+        params: dict = {
```

Comment:
You don't usually need to annotate variables inside methods/functions unless mypy complains :)